tests/fixtures_server.py

```diff
@@ -531,10 +531,11 @@ async def __job_factory(
         args.update(**kwargs)
         job = ApplyWorkflow(**args)
         db.add(job)
+        await db.commit()
```

Comment:
Let's check whether this diff is needed.
If not: let's remove it.
If yes: let's understand the reason better.